HW_08/task4.py

```diff
@@ -0,0 +1,11 @@
+# str, int, float, hex data
+lst = ["abc", "44", "0", 3, 1.5, .2, "2.2", "44", 2e1, 0x11, "4*7"]
+
+def digit_only(symbol):
+    if type(symbol) == int or type(symbol) == float:
```

Comment:
nit: nice function for checking type `isinstance()`

Reply:
Changed the verification:
if isinstance(symbol, (float, int)): 

Made change in branche and will make new pool request